data-catalog-backend-app/src/main/resources/db/migration/V1.0__initialize_schema_and_tables.sql

```diff
@@ -32,3 +30,70 @@ CREATE TABLE IF NOT EXISTS BACKEND_SCHEMA.INFORMATION_TYPE
     LAST_MODIFIED_BY     VARCHAR(200),
     LAST_MODIFIED_DATE   TIMESTAMP
 );
+
+CREATE TABLE IF NOT EXISTS BACKEND_SCHEMA.DATASET
+(
+    DATASET_ID           UUID PRIMARY KEY,
+    JSON_PROPERTY        JSONB        NOT NULL,
+    ELASTICSEARCH_STATUS TEXT         NOT NULL,
+    CREATED_BY           VARCHAR(200) NOT NULL,
+    CREATED_DATE         TIMESTAMP    NOT NULL,
+    LAST_MODIFIED_BY     VARCHAR(200),
+    LAST_MODIFIED_DATE   TIMESTAMP
+);
+
+CREATE TABLE IF NOT EXISTS BACKEND_SCHEMA.DISTRIBUTION_CHANNEL
+(
+    DISTRIBUTION_CHANNEL_ID UUID PRIMARY KEY,
+    NAME                    VARCHAR(100) UNIQUE NOT NULL,
+    DESCRIPTION             TEXT                NOT NULL,
+    PRODUCERS               TEXT                NOT NULL,
+    CONSUMERS               TEXT                NOT NULL,
```

Comment:
PRODUCERS               TEXT                NOT NULL,
    CONSUMERS               TEXT                NOT NULL,

Skal vel bort

Reply:
Fixed